services/TwilioProgrammableVideo/index.md

```diff
@@ -14,101 +14,96 @@ lastupdated: "2017-07-24"
-- DO include the basic, most-common-use scenario steps to use the service or integrate it into the app. 
-- DO NOT include steps to add the service from the Bluemix catalog; we assume that the user already took steps in the UI to add the service. 
-- DO include code snippets in all languages that can be copied, as well as VCAP service info.  
-- For additional tasks like configuring, managing, etc., add a task section (## Gerund_task_title) below the task section or "About" section if used. Use a task title such as "Configuring x", "Administering y", "Managing z". -->
+1. In the following screen, enter your `Account SID` and `Auth Token` from the
+   Twilio Console. In the Twilio Console, your credentials can be found here:
 
-<!-- You can include an optional prerequisites paragraph for any prerequisites to be met before integrating the service. For example: -->
+   ![Configure your Twilio Credentials](images/02-twilio-credentials.png)
 
-Before an application developer can embed single sign-on capability into an app, the administrator must create unbound service instances by using the Bluemix user interface.
+1. Navigate to the API Key Creation Screen in the Twilio Console. Create a new
+   API Key, saving both the `key` and the `secret` - you won’t be able to get it back again!
 
-<!-- Include a sentence to briefly introduce the steps. Examples: -->
+1. In the Bluemix Console, select 'Catalog' -> 'Apps' -> 'Cloud Foundry Apps'
+   then select the `SDK for Node.js`. Name it something memorable.
 
-To integrate your app with the service, complete these steps: -OR-
-To get up and running quickly with this service, follow these steps: -OR-
-Complete these steps to get started with the BigInsights service:
+1. In the Bluemix Dashboard, click on your Twilio Service under 'Services',
+   then click the 'Create connection +' button. Connect it to your new Node.js
+   App.
 
-<!-- Use ordered list markup for the step section. For code examples: 
-- use three backticks ahead of and after the example (```)
-- For copyable code snippet, multi-line, include {: codeblock} following the last set of backticks. A copy button will display in framework in output.
-- For copyable command, single line, include {: pre} following the last set of backticks. When displayed, it will show "$" at the beginning of the command example and a copy button, but the copy button will include just the command example.
-- For non-copyable output snippet, include {: screen} following the last set of backticks.
- -->
+1. Navigate back to your Bluemix app and click ‘Runtime’ on the left sidebar.
+   In environment variables, set two environment variables. Copy and paste the
+   APP Sid and a Voice-capable phone number:
 
-1. Step 1 to integrate app with the service.
-2. Step 2 to integrate app with the service.
+    ```bash
+    TWILIO_TWIML_APP_SID
+    TWILIO_NUMBER
+    ```
```

Comment:
@joliveros don't forget to append `{: screen}` below ```
like:

    ```bash
    TWILIO_TWIML_APP_SID
    TWILIO_NUMBER
    ```
    {: screen}